### PR TITLE
add procedure to include metadata in FieldSet

### PR DESCRIPTION
**Pull request by mo-lormi**
## Description
<br>

This PR is intended to add a procedure to include _metadata_ in FieldSet.
<br><br>

### Acceptance Criteria (Definition of Done)

All the tests pass.
<br><br>

### CODE REVIEW

File: src/atlas/field/FieldSet.cc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2013 ECMWF.
+ * (C) Copyright 2013-2023 ECMWF.
```
Comment (wdeconinck) [inline src/atlas/field/FieldSet.cc:2]: Not necessary to add "2023" (here and other file). We assume an open-ended approach.

Comment (mo-lormi) [reply src/atlas/field/FieldSet.cc]: ... done

**Review by wdeconinck — APPROVED** [review]
Perfect, Thanks!

**Comment by mo-lormi** [comment]
> Perfect, Thanks!

Thanks ...

**Pull request merged**